Makefile

```diff
@@ -1,6 +1,9 @@
 edit:
-	cd stockfish && $(MAKE) build ARCH=$(ARCH)
-	cd cutechess/gui && $(MAKE)
+	cd stockfish && $(MAKE)
```

Comment:
Does this work? When I tried this it complained about not having the ARCH variable

Reply:
Yeah I just pulled down your branch and the Makefile doesn't work for me

Reply:
Oops!  I must have accidentally removed the 'build ARCH=$(ARCH)'.  My latest commit fixes it